### PR TITLE
logging.py: Remove unnecessary import

### PR DESCRIPTION
**Pull request by anukul**
Removes duplicate import statement

Closes https://github.com/coala/coala/issues/3592

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

File: coalib/output/Logging.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from datetime import datetime
+import sys
 import json
 import io
 import logging
```
Comment (nemaniarjun) [inline coalib/output/Logging.py]: Should be alphabetical.

Comment (anukul) [inline coalib/output/Logging.py]: Should I reorder the whole list, then?

**Comment by nemaniarjun** [comment]
Please follow the [commit guidelines](http://coala.io/commit)   :)

File: coalib/output/Logging.py (hunk 1 above)
Comment (nemaniarjun) [inline coalib/output/Logging.py]: The whole list is already alphabetical. Just your placement to sys is wrong.

Comment (anukul) [inline coalib/output/Logging.py]: I'm not sure I understand.
Why does "io" occur after "json" and before "logging"?

Comment (nemaniarjun) [inline coalib/output/Logging.py]: My bad.

**Comment by Mixih** [comment]
@anukul Please correct your body to be grammatically correct. You may write in any tense you want.

File: coalib/output/Logging.py [hunk 2]
```diff
@@ -1,16 +1,15 @@
 from datetime import datetime
-import json
 import io
+import json
 import logging
 import logging.config
+import sys
```
Comment (Mixih) [inline coalib/output/Logging.py]: Why this move?

Comment (anukul) [inline coalib/output/Logging.py]: https://github.com/coala/coala/pull/3622#discussion_r97488525
Should I change it back?

**Comment by Udayan12167** [comment]
You have a merge commit. Please remove it.

**Comment by Udayan12167** [comment]
In your commit message body please make removed->Removes and make it a proper sentence.

**Comment by Udayan12167** [comment]
You should rebase your PR to master.

**Comment by jayvdb** [comment]
ack b87328e

**Comment by jayvdb** [comment]
Please rebase again.

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3622/commits/003704520ba6ecdda2da0c3cad150d82d66e20d0

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
Hey, this is your first contribution, right? Congrats and welcome to the coalaians! :) You've done your first step to become a coala developer.

Rultor will now automatically release your changes as a prerelease so users can experience the goodness of your changes right now.

We would really appreciate it if you could help us with

- [telling us about your contribution experience and what we can do better](http://coala.io/newform)
- [some code reviews](http://coala.io/reviewing) for other newcomers
- fixing at least [a difficulty/low issue](http://coala.io/low)
- fill our [usability survey](http://coala.io/usability)

If you've done those and tell us about it so we can promote you to a coala developer!

We're all volunteers and we're currently struggling to keep this up. Helping all those newcomers is a lot of work and we do need your help so we can continue this for others!

**Pull request merged**